spec/integration/operations_spec.rb

```diff
@@ -265,15 +265,15 @@ module Hawkular::Operations::RSpec
       expect(actual_data['driverName']).to eq(driver_name)
     end
 
-    it 'Redeploy should be performed and eventually respond with success' do
+    it 'Restart should be performed and eventually respond with success' do
       wf_server_resource_id = 'Local~~'
       alerts_war_resource_id = 'Local~%2Fdeployment%3Dhawkular-alerts-actions-email.war'
       path = CanonicalPath.new(tenant_id: @tenant_id,
                                feed_id: @feed_id,
                                resource_ids: [wf_server_resource_id, alerts_war_resource_id])
 
       redeploy = {
-        operationName: 'Redeploy',
+        operationName: 'Restart',
```

Comment:
After some meetings with product management the PRD was updated with the desired language around deployments.  The operations we'll offer are (WF terms in parens):  Deploy (add content), Undeploy (remove), Restart (redeploy), Enable (deploy) and Disable (undeploy).  I wanted to carry those terms through our code consistently from MIQ to gem to hawkular-services (command-gateway) to agent.